docs/griptape-cloud/data-sources/create-data-source.md

```diff
@@ -0,0 +1,9 @@
+# Data Sources
+
+Use [Data Sources](https://cloud.griptape.ai/data-sources) to connect and ingest your own data for use with your AI applications via [Knowledge Bases](../knowledge-bases/create-knowledge-base.md).
+
+## Create a Data Source
```

Comment:
I could understand if we're worried about things getting out of date, but I'd love to see some screenshots for anything that references actions you can take in the UI.

This would be particularly nice on the sections that are relatively sparse, such as the `Create a Data Source` section, which basically says "you can do this in the ui", and doesn't even list which data sources are supported. I assume that not listing data sources might be intentional, but without examples, I'm left with a slightly vague understanding of what a Data source is/could be.

Reply:
I avoided screenshots as they are the first things to go stale. I'll flesh this out a bit more with what Data Sources we support.